service/history/historyEngine.go

```diff
@@ -1536,6 +1579,42 @@ func (e *historyEngineImpl) SignalWorkflowExecution(signalRequest *h.SignalWorkf
 		})
 }
 
+func (e *historyEngineImpl) DeleteWorkflowExecutionSignal(deleteRequest *h.DeleteWorkflowExecutionSignalRequest) error {
+	domainID, err := getDomainUUID(deleteRequest.DomainUUID)
```

Comment:
Lets use the updateWorkflowExecution helper defined in historyEngineImpl for making this update.

Reply:
Changed.